components/src/components/hubspot-form/hubspot-form.tsx

```diff
@@ -46,6 +46,39 @@ export class HubspotForm {
             });
             this.isLoading = false;
             this.didLoad = true;
+
+            // Hidden form fields from HubSpot created forms causes gaps to appear in
+            // the form which is less then desireable. So we need to find the hidden
+            // form fields and then hide the parent <fieldset> for it. We will also need
+            // to populate those fields with the correct values.
+            waitForElementToExist(`#hubspotForm_${this.formId} form fieldset div`).then(() => {
+                const fieldSets = document.querySelectorAll(`#hubspotForm_${this.formId} form fieldset div[style*="display:none"]`);
```

Comment:
There's some duplication here that could probably be pulled into a variable.

Reply:
👍